website/cue/reference/remap/functions/get_vector_timezone.cue

```diff
@@ -0,0 +1,47 @@
+package metadata
+
+remap: functions: get_vector_timezone: {
+	category:    "System"
+	description: """
+		Returns the name of the timezone in the vector configuration
```

Comment:
```suggestion
		Returns the name of the timezone in the Vector configuration.
```

Reply:
I've changed to `Vector` -- note the parenthetical continuation of the sentence on the next line, so I kept the period after the clause in parens, see what you think.